airline-data/src/main/scala/com/patson/data/Constants.scala

```diff
@@ -64,6 +64,7 @@ object Constants {
   val AIRLINE_CASH_FLOW_ITEM_TABLE = "airline_cash_flow_item"
   val AIRLINE_BASE_SPECIALIZATION_TABLE = "airline_base_specialization"
   val AIRLINE_REPUTATION_BREAKDOWN = "airline_reputation_breakdown"
+  val BALANCE_AUDIT = "balance_audit"

```

Comment:
small nit - let's name this val `BALANCE_AUDIT_TABLE`, most vals follow this naming convention, i might as well fix the `AIRLINE_REPUTATION_BREAKDOWN` lol